engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -114,9 +115,14 @@ object WorkflowActor {
     */
   case class WorkflowActorData(currentLifecycleStateActor: Option[ActorRef],
                                workflowDescriptor: Option[EngineWorkflowDescriptor],
+                               initializationData: AllBackendInitializationData,
                                lastStateReached: StateCheckpoint)
   object WorkflowActorData {
-    def empty = WorkflowActorData(currentLifecycleStateActor = None, workflowDescriptor = None, StateCheckpoint(WorkflowUnstartedState))
+    def empty = WorkflowActorData(
+      currentLifecycleStateActor = None,
+      workflowDescriptor = None,
+      initializationData = AllBackendInitializationData.empty,
+      StateCheckpoint(WorkflowUnstartedState))
```

Comment:
I can see this isn't really your code, but it'd be nice to put `lastStateReached = StateCheckpoint(WorkflowUnstartedState)` to match all the others